src/udev/net/link-config.c

```diff
@@ -432,6 +433,16 @@ int link_config_apply(link_config_ctx *ctx, link_config *config,
         if (r < 0)
                 return log_device_warning_errno(device, r, "Could not find ifindex: %m");
 
+        DeviceAction a;
+        r = device_get_action(device, &a);
+        if (r < 0)
+                return log_device_warning_errno(device, r, "Failed to get ACTION= property: %m");
+
+        if (a != DEVICE_ACTION_ADD) {
```

Comment:
as mentioned, I think DEVICE_ACTION_BIND might be an action type where we want to call this too